src/game/common/language.h

```diff
@@ -438,7 +438,7 @@ enum TextEnum
     TXT_BADGER = 400, // Badger Bomber
     TXT_MIG = 401, // Mig Attack Plane
     TXT_YAK = 402, // Yak Attack Plane
-    TXT_FENC = 403, // Barbed Wire
+    TXT_FENCE = 403, // Barbed Wire
```

Comment:
This change should really be a separate PR or at least a separate commit as its unrelated to the fixing of Wall_Update.